doc/holochain_101/src/faq.md

```diff
@@ -1,27 +1,25 @@
- [What is Holochain _not_ good for?](#what-is-holochain-not-good-for)
-5. [What is Holochain's consensus algorithm?](#what-is-holochains-consensus-algorithm)
-6. [Can you run a cryptocurrency on Holochain?](#can-you-run-a-cryptocurrency-on-holochain)
-7. [How is Holochain different from __________?](#how-is-holochain-different-from-__________)
-8. [What language is Holochain written in? What languages can I use to make Holochain apps?](#what-language-is-holochain-written-in-what-languages-can-i-use-to-make-holochain-apps)
-9. [Is Holochain open source?](#is-holochain-open-source)
-10 [How is Holochain more environmentally ethical than blockchain?](#how-is-holochain-more-environmentally-ethical-than-blockchain)
-11. [How are data validated on Holochain?](#how-are-data-validated-on-holochain)
-12. [What happens to data when a node leaves the network?](#what-happens-to-data-when-a-node-leaves-the-network)
-13. [Should I build my coin/token on Holochain?](#should-i-build-my-cointoken-on-holochain)
-14. [What does “agent-centric” mean? How is this different from “data-centric”?](#what-does-agent-centric-mean-how-is-this-different-from-data-centric)
-15. [What is the TPS (Transactions Per Second) on Holochain?](#what-is-the-tps-transactions-per-second-on-holochain)
-
-
+1.  [How is Holochain different from blockchain?](#how-is-holochain-different-from-blockchain)
+2.  [Why do you call it "Holochain"?](#why-do-you-call-it-holochain)
+3.  [How is Holochain different from a DHT (Distributed Hash Table)?](#how-is-holochain-different-from-a-dht-distributed-hash-table)
+4.  [What kind of projects is Holochain good for?](#what-kind-of-projects-is-holochain-good-for)
+    [What is Holochain _not_ good for?](#what-is-holochain-not-good-for)
+5.  [What is Holochain's consensus algorithm?](#what-is-holochains-consensus-algorithm)
+6.  [Can you run a cryptocurrency on Holochain?](#can-you-run-a-cryptocurrency-on-holochain)
+7.  [How is Holochain different from \***\*\_\_\*\***?](#how-is-holochain-different-from-__________)
+8.  [What language is Holochain written in? What languages can I use to make Holochain apps?](#what-language-is-holochain-written-in-what-languages-can-i-use-to-make-holochain-apps)
+9.  [Is Holochain open source?](#is-holochain-open-source)
+    10 [How is Holochain more environmentally ethical than blockchain?](#how-is-holochain-more-environmentally-ethical-than-blockchain)
+10. [How are data validated on Holochain?](#how-are-data-validated-on-holochain)
+11. [What happens to data when a node leaves the network?](#what-happens-to-data-when-a-node-leaves-the-network)
+12. [Should I build my coin/token on Holochain?](#should-i-build-my-cointoken-on-holochain)
+13. [What does “agent-centric” mean? How is this different from “data-centric”?](#what-does-agent-centric-mean-how-is-this-different-from-data-centric)
+14. [What is the TPS (Transactions Per Second) on Holochain?](#what-is-the-tps-transactions-per-second-on-holochain)
```

Comment:
Could you change all these to `1.`
The markdown renderer is taking care of enumeration itself

Reply:
Thank you for commenting.  This work in progress is muddy due to the unanticipated effects of my javascript linter.  I plan to recommit these changes while keeping an eye on that.